modules/gdscript/gdscript_compiler.cpp

```diff
@@ -1072,10 +1072,8 @@ GDScriptCodeGenerator::Address GDScriptCompiler::_parse_expression(CodeGen &code
 				}
 
 				if (has_setter) {
-					if (!is_in_setter) {
-						// Store stack slot for the temp value.
-						target = codegen.add_temporary(_gdtype_from_datatype(assignment->assignee->get_datatype()));
-					}
+                    // Store stack slot for the temp value.
+                    target = codegen.add_temporary(_gdtype_from_datatype(assignment->assignee->get_datatype()));
```

Comment:
Indentation should be made using tabs, not spaces.

While not required, I recommend installing pre-commit hooks to get formatting to be done automatically: https://docs.godotengine.org/en/stable/community/contributing/code_style_guidelines.html#pre-commit-hook

Reply:
Oh, thanks. Corrected.